composer.json

```diff
@@ -22,9 +22,9 @@
         "keboola/api-error-control": "^3.9",
         "keboola/billing-api-php-client": "^1.1",
         "keboola/configuration-variables-resolver": "^3.2",
-        "keboola/dockerbundle": "dev-odin-library-3",
+        "keboola/dockerbundle": "dev-erik-PS-3920",
```

Comment:
Až bude mergnuté toto tak se to může vrátit https://github.com/keboola/docker-bundle/pull/678